### PR TITLE
Weather data: time zones, fix fractional DOY, PAR groups, QAQC plots

### PR DESCRIPTION
**Pull request by bpbond**
Clean up a few code things; specify EST as time zone (looks like this is what NEON is using); move DOY calculation later in pipeline to avoid fractional values. Also show a simple QA/QC plot for PAR.

PAR

### CODE REVIEW

**Comment by bpbond** [comment]
Interesting. Here's the computation of PAR group using a simple `ntile(Daytime_PAR, 3)`:
PAR

And here, we do it by month:
PAR

Seems that we want the latter approach, right? That's what I'm committing.

File: R/sapflow-Rs.Rmd [hunk 1]
```diff
@@ -72,56 +72,99 @@ read_csv("../SERC/baseliner_data/control_Kest_yr.csv",
...
 read_csv("../SERC/con_licor_data_20191120.csv") %>%
+  mutate(Timestamp = force_tz(Timestamp, tz = "EST")) %>% 
   left_join(ports_lt, by = "Port") %>% 
   left_join(inventory, by = c("Tree" = "Sapflux"))-> rs_long
 
 # Load weather data
 read_csv("../SERC/met_data/SECPRE_30min.csv") %>% 
-  mutate(Timestamp = ymd_hms(endDateTime)) %>% 
+  mutate(Timestamp = ymd_hms(endDateTime, tz = "UTC")) %>% 
   rename(Precip = secPrecipBulk) %>%
```
Comment (stephpenn1) [inline R/sapflow-Rs.Rmd:90]: please add a `with_tz = "EST"` step for all weather data

File: R/sapflow-Rs.Rmd [hunk 2]
```diff
@@ -72,56 +72,99 @@ read_csv("../SERC/baseliner_data/control_Kest_yr.csv",
...
+  wx_dat
+```
 
+## Diagnostic plots for weather data
+
+```{r, weather-qaqc}
+# Monthly averages by hour for each variable
+wx_dat %>% 
+  gather(var, value, PAR, Precip, RH, Tair) %>% 
+  mutate(Timestamp = with_tz(Timestamp, "EST"),
```
Comment (stephpenn1) [inline R/sapflow-Rs.Rmd:141]: probably remove this line since we will do this above

File: R/sapflow-Rs.Rmd (hunk 1 above)
Comment (bpbond) [reply R/sapflow-Rs.Rmd:90]: Done

File: R/sapflow-Rs.Rmd (hunk 2 above)
Comment (bpbond) [reply R/sapflow-Rs.Rmd]: Done

**Comment by bpbond** [comment]
Changes made @stephpenn1

**Pull request merged**